### PR TITLE
fixes binoculars targetting

### PR DESCRIPTION
**Pull request by SplinterGP**
## About The Pull Request
could probably use signals instead of mouse intercept but lazy

## Changelog
:cl:
fix: binoculars now can laser target properly.
/:cl:

### CODE REVIEW

**Comment by BraveMole** [comment]
now fix firing on black tiles :)

**Comment by SplinterGP** [comment]
> 
> 
> now fix firing on black tiles :)

who broke it pepega

**Comment by BraveMole** [comment]
> > now fix firing on black tiles :)
> 
> who broke it pepega

Well... You?

File: code/game/objects/items/binoculars.dm [hunk 1]
```diff
@@ -82,41 +82,31 @@
...
-	return TRUE
-
-
-/obj/item/binoculars/tactical/on_unset_interaction(mob/user)
+/obj/item/binoculars/tactical/onunzoom(mob/living/user)
 	. = ..()
 
+	if(laser)
+		QDEL_NULL(laser)
+
```
Comment (TiviPlus) [inline code/game/objects/items/binoculars.dm:96]: Uneeeded if

Comment (SplinterGP) [reply code/game/objects/items/binoculars.dm:96]: > Uneeeded if

Suggest pls for webcommit

**Comment by SplinterGP** [comment]
> > > now fix firing on black tiles :)
> > 
> > who broke it pepega
> 
> Well... You?

When?

**Comment by SplinterGP** [comment]
You mean being able to laser on the black tiles? Thats intended

File: code/game/objects/items/binoculars.dm (hunk 1 above)
Comment (TiviPlus) [reply code/game/objects/items/binoculars.dm:96]:
```suggestion
	QDEL_NULL(laser)

```

Comment (TiviPlus) [reply code/game/objects/items/binoculars.dm:96]: noob

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**